frontend/src/pages/projects/screens/spawner/environmentVariables/EnvSecret.tsx

```diff
@@ -31,15 +29,6 @@ const EnvSecret: React.FC<EnvSecretProps> = ({ env = DEFAULT_ENV, onUpdate }) =>
           />
         ),
       },
-      [SecretCategory.AWS]: {
```

Comment:
In the issue description I saw
```
We should also replace AWS Secret with Upload Secret.
```
Should we replace this option with a generic Secret upload just the same as ConfigMap upload which is already implemented, instead of getting rid of it?

Reply:
Yup -- needs Secret Upload to replace this.